doc/man3/X509_VERIFY_PARAM_set_flags.pod

```diff
@@ -145,12 +145,16 @@ To obtain all names start with B<n> = 0 and increment B<n> as long as no NULL
 pointer is returned.
 
 X509_VERIFY_PARAM_set1_host() sets the expected DNS hostname to
-B<name> clearing any previously specified hostname.  If
-B<name> is NULL, or empty the list of hostnames is cleared, and
+B<name>, clearing any previously specified hostname.
+If B<name> is NULL or empty, the list of hostnames is cleared, and
 name checks are not performed on the peer certificate.  If B<name>
 is NUL-terminated, B<namelen> may be zero, otherwise B<namelen>
 must be set to the length of B<name>.
 
+TLS clients are recommended to use this function or X509_VERIFY_PARAM_add1_host()
```

Comment:
As above. I would not recommend this function but instead `SSL_set1_host()`

Reply:
Changed also this accordingly.